static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -157,6 +157,40 @@ define([
             return config.switches.mobileTopBannerRemove && $('.top-banner-ad-container--ab-mobile').length > 0 && detect.getBreakpoint() === 'mobile';
         },
 
+        isSponsorshipContainerTest = function () {
+            var sponsorshipIds = ['#dfp-ad--adbadge', '#dfp-ad--spbadge', '#dfp-ad--fobadge', '#dfp-ad--adbadge1', '#dfp-ad--spbadge1', '#dfp-ad--fobadge1', '#dfp-ad--adbadge2', '#dfp-ad--spbadge2', '#dfp-ad--fobadge2', '#dfp-ad--adbadge3', '#dfp-ad--spbadge3', '#dfp-ad--fobadge3', '#dfp-ad--adbadge4', '#dfp-ad--spbadge4', '#dfp-ad--fobadge4', '#dfp-ad--adbadge5', '#dfp-ad--spbadge5', '#dfp-ad--fobadge5'],
+                sponsorshipIdsReturned = [],
+                sponsorshipIdReturned;
+
+            _.forEach(sponsorshipIds, function (value) {
+                if ($(value).length) {
+                    sponsorshipIdReturned = value;
+                    sponsorshipIdsReturned.push(sponsorshipIdReturned);
```

Comment:
Do you need `sponsorshipIdReturned`? You can do just `sponsorshipIdsReturned.push(value);`